pyvista/core/filters/data_set.py

```diff
@@ -4613,6 +4613,76 @@ def shrink(self, shrink_factor=1.0, progress_bar=False):
             return output.extract_surface()
         return output
 
+    def tessellate(self, max_n_subdivide=3, merge_points=True, progress_bar=False):
+        """Tessellate a mesh.
+
+        This filter approximates nonlinear FEM-like elements with linear
+        simplices. The output mesh will have geometry and any fields specified
+        as attributes in the input mesh's point data. The attribute's copy
+        flags are honored, except for normals.
+
+        For more details see `vtkTessellatorFilter <https://vtk.org/doc/nightly/html/classvtkTessellatorFilter.html#details>`_.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
```

Comment:
```suggestion
        progress_bar : bool, optional
            Display a progress bar to indicate progress.

```

Reply:
Sorry @tkoyama010 I think I got this suggestion wrong. I meant to delete this duplicate block but I forgot to actually delete it in the suggestion :roll_eyes:

Reply:
Oh, I see, the mystery of not being able to commit has been solved 😂
```suggestion
```